src/views/app/folder-panel/lists-item/message-list-item.jsx

```diff
@@ -207,16 +206,62 @@ export default function MessageListItem({
 									wrap="nowrap"
 									takeAvailableSpace
 									mainAlignment="flex-start"
-									crossAlignment="baseline"
+									crossAlignment="center"
 								>
+									{item.isSentByMe && !item.isDraft && !item.isReplied && !item.isForwarded && (
+										<Tooltip label={t('label.sent', 'Sent')} placement="bottom">
+											<Padding right="extrasmall">
+												<Icon data-testid="SentIcon" icon="PaperPlaneOutline" color="secondary" />
+											</Padding>
+										</Tooltip>
+									)}
+									{item.isDraft && (
+										<Tooltip label={t('label.draft', 'Draft')} placement="bottom">
+											<Padding right="extrasmall">
+												<Icon data-testid="DraftIcon" icon="FileOutline" color="secondary" />
+											</Padding>
+										</Tooltip>
+									)}
+									{item.isReplied && (
+										<Tooltip label={t('label.replied', 'Replied')} placement="bottom">
+											<Padding right="extrasmall">
+												<Icon data-testid="RepliedIcon" icon="UndoOutline" color="secondary" />
+											</Padding>
+										</Tooltip>
+									)}
+									{item.read === false &&
+										!item.isReplied &&
+										!item.isDraft &&
+										!item.isSentByMe &&
+										!item.isForwarded && (
+											<Tooltip label={t('search.unread', 'Unread')} placement="bottom">
+												<Padding right="extrasmall">
+													<Icon data-testid="UnreadIcon" icon="EmailOutline" color="secondary" />
+												</Padding>
+											</Tooltip>
+										)}
```

Comment:
you can optimise this by creating a memorised variable like 
```js
const showUnreadIcon =useMemo(()=> item.read === false &&
										!item.isReplied &&
										!item.isDraft &&
										!item.isSentByMe &&
										!item.isForwarded ,[item])
```
or even better if you write some function to dynamically calculate the icon, label and tooltip and replace the all these condition with something like
```js
{ showIcon && <Tooltip label={tooltipLabel} placement="bottom">
												<Padding right="extrasmall">
													<Icon data-testid="UnreadIcon" icon={icon} color="secondary" />
												</Padding>
											</Tooltip> }
```